src/elements/content-sidebar/SidebarSection.js

```diff
@@ -1,3 +1,5 @@
+// @deprecated, use Collapsible
```

Comment:
Are we sure this is correct? There are multiple uses of this component that don't collapse.

Reply:
This was meant to be replaced by Collapsible since that was created using code from here + additional stuff. But we never went back and nuked this one out.